packages/bpk-component-segmented-control/src/BpkSegmentedControl.module.scss

```diff
@@ -102,3 +102,19 @@
 .bpk-segmented-control[class*='rightOfOption'] {
   border-inline-start: none;
 }
+
+.bpk-segmented-control:first-child {
+  border-top-left-radius: tokens.$bpk-border-radius-sm;
+  border-bottom-left-radius: tokens.$bpk-border-radius-sm;
+}
+
+.bpk-segmented-control:last-child {
+  border-top-right-radius: tokens.$bpk-border-radius-sm;
+  border-bottom-right-radius: tokens.$bpk-border-radius-sm;
+}
+
+.bpk-segmented-control:focus {
+  border: (tokens.$bpk-one-pixel-rem * 2) solid tokens.$bpk-text-link-day;
+  outline: none;
+  outline-offset: (tokens.$bpk-one-pixel-rem * 2);
+}
```

Comment:
Setting borders the outer children to ensure the focus isn't clipped off the corners.

Adding outline none is to ensure this extra outline doesn't show

<img width="327" alt="Screenshot 2024-07-16 at 14 57 13" src="https://github.com/user-attachments/assets/02a18aca-eb5e-4256-8cef-ac348cd0f2ec">